cinder/files/mitaka/cinder.conf.controller.Debian

```diff
@@ -39,7 +40,7 @@ volume_backend_name=DEFAULT
 
 default_volume_type={{ controller.default_volume_type }}
 
-enabled_backends={% for backend_name, backend in controller.get('backend', {}).iteritems() %}{{ backend_name }}{% if not loop.last %},{% endif %}{% endfor %}
+enabled_backends={% controller.backend|join(',') %}
```

Comment:
I'd not go this way, as we move towards having optional backend.name as configurable name